fingerprint_verification.py

```diff
@@ -6,39 +6,103 @@
 
 def read_print(filename):
     """Reads the given print and returns a dictionary with all info."""
-    # Your turn!
-
-
-
-
-
-
-# You can add your various check functions below
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+    fingerprint_data = {}
+    with open(filename) as fh:
+        name = fh.readline().strip()
+        width = int(fh.readline().strip())
+        height = int(fh.readline().strip())
+        
+        fingerprint_data['name'] = name
+        fingerprint_data['width'] = width
+        fingerprint_data['height'] = height
+
+        fingerprint = [] 
+
+        for line in filename:
+            cleaned_line = line.rstrip()  # Keep trailing spaces but remove newlines
```

Comment:
So the danger of `rstrip` here over just slicing off the last character (the newline) is that all of your strings are going to be different lengths, which will make indexing more of a pain later.

Alternatively, if you were worried about the fact that the last line won't have a newline character, you could always use `.replace` to just replace the newline character in each line with an empty string.